src/sinks/aws_sqs.rs

```diff
@@ -148,25 +185,23 @@ impl SinkConfig for SqsSinkConfig {
 impl SqsSinkConfig {
     pub async fn healthcheck(self, client: SqsClient) -> crate::Result<()> {
         client
-            .get_queue_attributes(GetQueueAttributesRequest {
-                attribute_names: None,
-                queue_url: self.queue_url.clone(),
-            })
+            .get_queue_attributes()
+            .queue_url(self.queue_url.clone())
+            .send()
             .await
             .map(|_| ())
             .context(GetQueueAttributesSnafu)
             .map_err(Into::into)
     }
 
-    pub fn create_client(&self, proxy: &ProxyConfig) -> crate::Result<SqsClient> {
-        let region = (&self.region).try_into()?;
-        let tls_settings = MaybeTlsSettings::from(TlsSettings::from_options(&self.tls)?);
```

Comment:
I don't see `self.tls` referenced any more, is that intentional? Can we drop the `tls` field from `SqsSinkConfig`?

Reply:
Oof, good catch. We should preserve the `tls` settings. Users do use that when using AWS API-compatible tools. I think this needs to be fixed for the other AWS components that were migrated, as well. We can probably just add it as another argument to `create_client`?

Reply:
I'll look at this more. I might need to add support for these in the client builder